Add team config (#219)
seek-oss/logger: docs/eeeoh.md

  eeeoh: {
    datadog: 'tin',
    use: 'environment',

>     team: 'my-team-name', // Optional

Reviewer: We can probably over-index on SEEK-isms here:

```suggestion
    team: 'my-owner-name', // Optional
```

  },
});
```

The `use: 'environment'` option is the recommended approach of sourcing application metadata from the workload hosting environment to annotate logs.

> The `team` option is optional and can be used to set the name of the team responsible for the component.

Reviewer: ```suggestion
The `team` option is optional and can be used to set the name of the team that owns the component or specific log.
```

import { createLogger } from '@seek/logger';

const logger = createLogger({

>   eeeoh: { datadog: 'tin', use: 'environment', team: 'my-team-name' },

Reviewer: ```suggestion
  eeeoh: { datadog: 'tin', use: 'environment', team: 'my-owner-name' },
```

});
```

### By team

> If your service is shared by multiple teams, you can create a separate logger per team.

Reviewer: Not sure if this is any clearer:

```suggestion
If your service emits logs that correspond to different teams,
you can create a separate logger per team.
```
